README.md

```diff
@@ -189,7 +180,7 @@ Next is to run a few DL models to validate the end-to-end functionality.
         --data_dir <input_data_location> 
     ```
 
-## OS X Instructions
+## OS X instructions
```

Comment:
add: "build instructions"

Reply:
Still not accurate of a description though.  So I reworded it slightly... lmk what you think